.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
not super important here, but it's pretty typical to add `.DS_Store` to this file so you're not tracking changes to the folder metadata from Mac OS